age-app/src/app/filling-container.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+import React from "react";
+import { useState } from "react";
+import "./filling-container.css";
+import FillingBox from "./filling-box";
+export default function FillingContainer() {
+  //store input day, month, year
+  const [day, setDay] = useState("");
+  const [month, setMonth] = useState("");
+  const [year, setYear] = useState("");
```

Comment:
Instead of making many individual states, you can make one state object that contains all data you need.